integration-tests/common/common.go

```diff
@@ -1,349 +1,127 @@
-	for i := range bootstrapPeers {
-		p2pBootstrappers = append(p2pBootstrappers, bootstrapPeers[i].P2PV2Bootstrapper())
-	}
-
-	// TODO: convert contractNodeInfo.BootstrapBridgeInfo.RelayConfig
-	// and contractNodeInfo.BridgeInfos[nIdx].RelayConfig,
-	relayConfig := job.JSONConfig{}
-
-	oracleSpec := job.OCR2OracleSpec{
-		ContractID:         contractNodeInfo.OCR2Address,
-		Relay:              ChainName,
-		RelayConfig:        relayConfig,
-		PluginType:         "median",
-		P2PV2Bootstrappers: pq.StringArray{strings.Join(p2pBootstrappers, ",")},
-		OCRKeyBundleID:     null.StringFrom(contractNodeInfo.BootstrapNodeKeysBundle.OCR2Key.Data.ID),
-		TransmitterID:      null.StringFrom(contractNodeInfo.BootstrapNodeKeysBundle.TXKey.Data.ID),
-		PluginConfig: job.JSONConfig{
-			"juelsPerFeeCoinSource": contractNodeInfo.BootstrapBridgeInfo.JuelsSource,
-		},
-	}
-	jobSpec := &client.OCR2TaskJobSpec{
-		Name:              fmt.Sprintf("terra-OCRv2-%s-%s", "bootstrap", uuid.NewV4().String()),
-		JobType:           "bootstrap",
-		OCR2OracleSpec:    oracleSpec,
-		ObservationSource: contractNodeInfo.BootstrapBridgeInfo.ObservationSource,
-	}
-	if _, err := contractNodeInfo.BootstrapNode.MustCreateJob(jobSpec); err != nil {
-		return fmt.Errorf("failed creating job for boostrap node: %w", err)
-	}
-	for nIdx, n := range contractNodeInfo.Nodes {
-		oracleSpec := job.OCR2OracleSpec{
-			ContractID:         contractNodeInfo.OCR2Address,
-			Relay:              ChainName,
-			RelayConfig:        relayConfig,
-			PluginType:         "median",
-			P2PV2Bootstrappers: pq.StringArray{strings.Join(p2pBootstrappers, ",")},
-			OCRKeyBundleID:     null.StringFrom(contractNodeInfo.NodeKeysBundle[nIdx].OCR2Key.Data.ID),
-			TransmitterID:      null.StringFrom(contractNodeInfo.NodeKeysBundle[nIdx].TXKey.Data.ID),
-			PluginConfig: job.JSONConfig{
-				"juelsPerFeeCoinSource": contractNodeInfo.BridgeInfos[nIdx].JuelsSource,
-			},
-		}
-		jobSpec := &client.OCR2TaskJobSpec{
-			Name:              fmt.Sprintf("terra-OCRv2-%d-%s", nIdx, uuid.NewV4().String()),
-			JobType:           "offchainreporting2",
-			OCR2OracleSpec:    oracleSpec,
-			ObservationSource: contractNodeInfo.BridgeInfos[nIdx].ObservationSource,
-		}
-		if _, err := n.MustCreateJob(jobSpec); err != nil {
-			return fmt.Errorf("failed creating job for node %s: %w", n.URL(), err)
-		}
-	}
-	return nil
+	return c
 }
 
-func BuildNodeContractPairID(node *client.Chainlink, ocr2Addr string) (string, error) {
-	csaKeys, _, err := node.ReadCSAKeys()
-	if err != nil {
-		return "", err
-	}
-	shortNodeAddr := csaKeys.Data[0].Attributes.PublicKey[2:12]
-	shortOCRAddr := ocr2Addr[2:12]
-	return strings.ToLower(fmt.Sprintf("node_%s_contract_%s", shortNodeAddr, shortOCRAddr)), nil
+func (c *Common) Default(t *testing.T) {
+	c.K8Config = &environment.Config{NamespacePrefix: "chainlink-ocr-cosmos", TTL: c.TTL, Test: t}
+	// These can be uncommented when toml configuration is supposrted for cosmos in the chainlink node
```

Comment:
This will need to be fixed since the chainlink node currently does not accept the toml for cosmos.

Reply:
We can build against https://github.com/smartcontractkit/chainlink/pull/8567 which will be merged into core soon